lib/srv/db/postgres/engine.go

```diff
@@ -202,7 +202,7 @@ func (e *Engine) handleStartup(client *pgproto3.Backend, sessionCtx *common.Sess
 	}
 	switch m := startupMessageI.(type) {
 	case *pgproto3.StartupMessage:
-		e.Log.Debugf("Received startup message: %#v.", m)
+		e.Log.DebugContext(e.Context, "Received startup message.", "message", fmt.Sprintf("%#v", m))
```

Comment:
```suggestion
		e.Log.DebugContext(e.Context, "Received startup message.", "message", m)
```
slog prints out the go fields by default.

Reply:
TIL.